lib/puppet/parser/functions/ensure_php_packages.rb

```diff
@@ -0,0 +1,35 @@
+module Puppet::Parser::Functions
+  newfunction(:ensure_php_packages, :type => :statement) do |arguments|
```

Comment:
Nitpick, but could we document the function and its arguments?

Reply:
will do!